src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
>The reason roots in RequestParser::parseFormData(), I think there is a bug here. Consider the same form data transmitted in application/x-www-form-urlencoded and multipart/form-data.

Then the bug is probably in `RequestParser::splitMultipartData()`. Can you do the following changes and test it?
```c++
ret << data.mid(start, end - start);
// replace with
ret << data.mid(start, (end - 1) - start);
// in 2 places inside RequestParser::splitMultipartData()
```